.github/workflows/ci.yml

```diff
@@ -0,0 +1,18 @@
+name: CI
+on:
+  push:
+  pull_request:
```

Comment:
There should be a branch filter here right? It seems wasteful to run for other pushes.